README.md

```diff
@@ -25,7 +27,7 @@ git commit -m 'your comment'
 git push origin [your_branch]
```

Comment:
Once the changes have been merged, do we need a clean-up step to remove the branch from origin?

Reply:
It depends on if the person who merges the PR uses GitHub's option to delete after merge. If not, then yes you are correct. I can add a comment, thanks.

Reply:
Updated